pages/_resources/fdspp-dhs-data-stewardship-tactical-wg.md

```diff
@@ -0,0 +1,125 @@
+---
+resource_name: Investing in Learning with the Data Stewardship Tactical Working Group at DHS
+slug: fdspp-dhs-data-stewardship-tactical-wg
+description: The Department of Homeland Security (DHS) experience forming the
+  Data Stewardship Tactical Working Group (DSTWG) provides meaningful insights
+  for those who want to address data-related challenges collaboratively and
+  successfully in their own agencies.
+source: Department of Homeland Security
+category: Case studies & examples
+tags:
+  - data governance
+  - data management
+  - Federal Data Strategy
+policy_tags:
+  - FDS Principle 03 Promote Transparency
+  - FDS Principle 08 Invest in Learning
+  - FDS Principle 09 Develop Data Leaders
+  - FDS Principle 10 Practice Accountability
+  - FDS Practice 02 Assess and Balance the Needs of Stakeholders
+format: ""
+details: >-
```

Comment:
This feels a bit clunky for frontmatter. Why not put it in the document body?

Reply:
Definitely clunky, but then again these files aren't intended to be edited by hand. Netlify will handle the saving/loading of these collection items for us and it likes to put everything in front matter. There might be a way to put the `details` in the body if it is necessary -- maybe something for the next team to look at.